src/main/archetype/dispatcher.ams/src/conf.d/proxy/cif-magento-graphql.proxy

```diff
@@ -0,0 +1,9 @@
+
+Include /etc/httpd/conf.d/variables/cif-magento-proxy.vars
+
+# Defines the proxy rule to access the Magento GraphQL endpoint
+
+<IfDefine ENABLE_CIF_MAGENTO_PROXY>
+	ProxyPass "/api/graphql" "${MAGENTO_HOST}/graphql" max=20 connectiontimeout=5 timeout=5 ttl=30
```

Comment:
Rename `COMMERCE_HOST`, since this can be used for non Magento as well. Also we should include `/graphql` into the variable, since this can be anything, for example the I/O Runtime GraphQl mapper is used.

Also see https://git.corp.adobe.com/Granite/dispatcher-k8s-base/blob/master/images/dispatcher/src/main/docker/container/root/etc/httpd/conf.d/dispatcher_vhost.conf#L72

Reply:
I included all your proposed changes, and also the one from @laurentiumagureanu as it's anyway nicer to have a `Location` block and this was also done like in AEMaaCS. I also added the removal of Cookies, we don't them for anything related to the GraphQL endpoint.